src/helpers/thunk.ts

```diff
@@ -1,5 +1,5 @@
-import {VNode, VNodeData} from './vnode';
-import {h} from './h';
+import {VNode, VNodeData} from '../types';
+import h from './h';
```

Comment:
What's the reasoning behind default exported `h`? I've been hearing complaints about them and a trend to use only named exports.

Reply:
I'm fine either way, we just need to pick one instead of exporting both!

As one of the champions of ES6 modules, the idea was to always favor default exports when possible, but one thing is what TC39 believes, and a very different thing is what folks actually do :)

Reply:
I agree that only exporting one is best. Personally, I prefer to avoid default exports but it's not a big deal to me.